source/whats-new.txt

```diff
@@ -58,6 +58,8 @@ The 5.0 driver release introduces the following features:
   ``Decimal128`` values. In particular, the error responses for the
   ``isNumber()`` and ``asNumber()`` methods match the Java responses for
   equivalent ``Decimal128`` values.
+  
+- This driver deprecates the ``org.mongodb.scala.Observable.completeWithUnit`` method
```

Comment:
S: Could be helpful to include what, if anything, users should use instead

Reply:
This is related to the breaking change about Observables - I've added a reference to the breaking change description here.